### PR TITLE
Add pages models and schemas

### PR DESCRIPTION
**Pull request by MaanasArora**
No tests added yet, but it would be appreciated if you could review it for any errors or changes.

I will add tests before this PR is merged.

Thank you!

### CODE REVIEW

File: lib/cursif/accounts/user.ex [hunk 1]
```diff
@@ -23,6 +25,8 @@ defmodule Cursif.Accounts.User do
     field :password, :string, virtual: true, redact: true
     field :hashed_password, :string, redact: true
 
+    has_many :pages, Page, foreign_key: :author_id
```
Comment (PenguinBoi12) [inline lib/cursif/accounts/user.ex]: A user doesn't have pages associated with him. The `belong_to` already does the job of associating the user and pages.

File: lib/cursif_web/schema.ex [hunk 2]
```diff
@@ -5,24 +5,30 @@ defmodule CursifWeb.Schema do
```
Comment (PenguinBoi12) [inline lib/cursif_web/schema.ex:5]: I've found a cleaner way, that also seems more idiomatic, to do the fields importation. You can check the notebook branch to see how [import](https://github.com/Code-Society-Lab/cursif/blob/notebook/lib/cursif_web/schema.ex) and their [definition](https://github.com/Code-Society-Lab/cursif/blob/notebook/lib/cursif_web/schema/notebook_types.ex).

**Review by PenguinBoi12 — CHANGES_REQUESTED** [review]
I am unsure if pages should have their own context or be added to the notebooks context (which doesn't exist yet on your branch and on the main branche). For some reason, my instinct tells me that page shouldn't have his own context. I'm saying that because I don't see `pages` be used outside of a notebook or at least out of a notebook context. 

To be faire, There's no notebook yet so don't worry about it. Beside, I am mostly thinking "out loud" and asking your opinion on this. What might simply happens is, I'll accept this and refactor it if I need to.

---

So far so good. The code seems pretty good. It would be nice to add the tests but otherwise it's awesome. Well done! Except for the test and, `has_many` and the absinthe schema thing, there's nothing else to change. 

Oh I would also change the name `contents` for `content`.

File: lib/cursif/accounts/user.ex (hunk 1 above)
Comment (MaanasArora) [reply lib/cursif/accounts/user.ex]: Done!

File: lib/cursif_web/schema.ex (hunk 2 above)
Comment (MaanasArora) [reply lib/cursif_web/schema.ex:5]: Done as well.

**Review by PenguinBoi12 — APPROVED** [review]
Has been merged with the notebook branch

**Pull request closed without merging**